### DISCUSSION TITLE
Duda

### DISCUSSION THREAD

**Discussion by juanoviedolx** [Launch-X-Latam/MisionFrontEnd#447] [Team Posts]
Una vez realizada cualquier practica, ¿se sube en algún sitio?

**Comment by MrAlizarinCrimson** [comment]
Buena pregunta, yo tampoco tengo idea, tampoco como es el tema del teams... Cómo se ingresa al grupo?

**Comment by juanoviedolx** [comment]
En el **OnBoarding**, en el apartado de **"Accede** al grupo de la Misión FrontEnd en Microsoft Teams" hay un enlace [Entra a tu grupo en Microsoft Teams](https://forms.office.com/r/Ddr5fxJbNk) le das click y te manda a un formulario, lo llenas con los datos de tu cuenta de Microsoft la cual te proporciono LaunchX, luego de llenarla pasa un tiempo para que te llegue un correo (revisa outlook) te llega como un "boleto" le das click, te muestra un mensaje de microsoft teams para que envies tu solicitud para unirte al grupo, después de un tiempo te la aceptan y ya entras.

**Comment by DaveOval** [comment]
aun no han dicho en donde, solo dijeron que en unas semanas pondran una seccion para mandar las practicas, lo que se ha estado haciendo es hacer un repositorio en tu perfil de github y ahi tenerlo listo para cuando digan a donde mandarlo

**Comment by juanoviedolx** [comment]
thx
